website/docs/quickstart_cli.md

```diff
@@ -0,0 +1,210 @@
+
+:::note
+
+If your FASTA is not indexed, you can use the `samtools` tool to index it.
+
+```sh-session
+samtools faidx volvox.fa
+# generates volvox.fa.fai
+```
+
+Or if you want to compress your FASTA, you can use `bgzip` as well.
+
+```sh-session
+bgzip volvox.fa
+samtools faidx volvox.fa.gz
+# compresses volvox.fa to volvox.fa.gz and generates volvox.fa.gz.fai and volvox.fa.gz.gzi
+```
+
+For more info about `bgzip` and `samtools`, see https://www.htslib.org/.
+
+:::
+
+If you have your JBrowse 2
+[running as described](quickstart_web#running-jbrowse-2) in the JBrowse web
+quickstart, you can refresh the page and an add a linear genome view. You will
+now see your config in the Assembly dropdown.
+
+![JBrowse 2 linear genome view setup with volvox in assembly dropdown](./img/lgv_assembly.png)
+
+## Adding a track
+
+Now we will show you how to add an alignments track and a variant track to
+JBrowse 2.
+
+:::info
+
+For this step we configure JBrowse to use files located on your computer. For an
+example of how to use files hosted at a URL, see the
+[adding a genome assembly](#adding-a-genome-assembly) step.
+
+:::
+
+### Adding an alignments track
+
+For this example we will use a BAM file to add an alignments track. Again, for
+this example we provide a small example BAM, but for your data you will replace
+the file name with the names of your data files.
+
+For this track we will assume the data is on your computer at the locations
+`/data/volvox.bam` and `/data/volvox.bam.bai`. You can download these file here
+if you want to run this example:
+[BAM](http://jbrowse.org.s3.amazonaws.com/genomes/volvox/volvox.bam) and
+[BAM index](http://jbrowse.org.s3.amazonaws.com/genomes/volvox/volvox.bam.bai).
+
+To add the track, run
+
+```sh-session
+# Replace with the location of your BAM file
+jbrowse add-track /data/volvox.bam --load copy
+```
+
+This will copy the BAM and BAM index into the JBrowse 2 directory and add a
+track pointing at those files to the config file. To see more options adding the
+track, such as specifying a name, run `jbrowse add-track --help`.
+
+If you don't want to copy your BAM file, you can use `--move` to move the file
+into the JBrowse 2 directory or `--symlink` to add a symlink to the file to the
+JBrowse 2 directory. If you want more control over the location, you can use
+`inPlace` to point the track at the file where it is, but be careful with this
+option because on a traditional server you will need to ensure that the file is
+in a place where the web server is serving it.
+
+:::note
+
+If your BAM is not indexed, you can use the `samtools` tool to index it.
+
+```sh-session
+samtools index volvox.bam
+# generates volvox.bam.bai
+```
+
+For more info about `samtools`, see https://www.htslib.org/.
+
+:::
+
+If you have your JBrowse 2
+[running as described](quickstart_web#running-jbrowse-2) in the JBrowse web
+quickstart, you can refresh the page and an add a linear genome view of the
```

Comment:
For these kind of descriptions, is it worth adding the path in the menu? I.e. add a linear genome view (`File -> Add -> Linear Genome View`)